sdk/java-sdk-testkit/src/main/scala/com/akkaserverless/javasdk/testkit/impl/TestKitEventSourcedEntityCommandContext.scala

```diff
@@ -30,6 +30,11 @@ final class TestKitEventSourcedEntityCommandContext(
     override val metadata: Metadata = Metadata.EMPTY)
     extends CommandContext
     with InternalContext {
+
+  def this(metadat: Metadata) {
+    this(metadata = metadat)
```

Comment:
```suggestion
  def this(metadata: Metadata) {
    this(metadata = metadata)
```

Reply:
Thanks for picking this up Johan but unfortunately this wouldn't work because the compiler gets an ambiguous reference for the constructor. Doesn't know if `this` refers to the very same constructor or the constructor with 5 parameters.